app/views/includes/taxs_main.scala.html

```diff
@@ -42,20 +42,16 @@
 @fullTitle = {@title - @Messages("generic.ats.browser.title")}
 
 @linkElement = {
-    <!--[if gt IE 8]><!--><link rel='stylesheet' media='screen' href='@controllers.routes.Assets.at("stylesheets/taxs.css")' /><!--<![endif]-->
-    <!--[if IE 6]><link rel='stylesheet' media='screen' href='@controllers.routes.Assets.at("stylesheets/taxs-ie6.css")' /><![endif]-->
-    <!--[if IE 7]><link rel='stylesheet' media='screen' href='@controllers.routes.Assets.at("stylesheets/taxs-ie7.css")' /><![endif]-->
-    <!--[if IE 8]><link rel='stylesheet' media='screen' href='@controllers.routes.Assets.at("stylesheets/taxs-ie8.css")' /><![endif]-->
-    <link rel='stylesheet' href='@controllers.routes.Assets.at("stylesheets/magna-charta/magna-charta.css")' />
-    <link rel='stylesheet' media='print' href='@controllers.routes.Assets.at("stylesheets/taxs-print.css")' />
+    <!--[if gt IE 8]><!--><link rel='stylesheet' media='screen' href='@controllers.routes.Assets.versioned("stylesheets/taxs.css")' /><!--<![endif]-->
```

Comment:
Haven't you removed all the IE stylesheets?

Reply:
Nope I deleted the ie7.scss/ie8.scss files which were probably not even getting used previously as it would get compiled in to one file if there was a sass compiler but funny thing is that there wasn't even a sass compiler so those SASS files were not doing anything so moved the styles in to ie7.css/ie8.css which I hope will work now

Reply:
Ok, great 👍